services/QuillLMS/app/models/subscription.rb

```diff
@@ -182,6 +182,10 @@ def is_trial?
     account_type && TRIAL_TYPES.include?(account_type)
   end
 
+  def is_school_or_district_premium?
```

Comment:
Note that a similar method is defined in User.rb, with subtly different business logic:

  ```
def school_premium?
    school&.subscription&.present?
  end

  def district_premium?
    school&.district&.subscription&.present?
  end

  def school_or_district_premium?
    school_premium? || district_premium?
  end
```